File: Circulo.java
public class Circulo extends Figuras{
    double radio;
    
    public Circulo(double radio){
        this.radio = radio;
    }

    @Override
    double area() {
        return Math.PI * (radio * radio);
    }

    @Override
    double perimetro() {
        return 2 * Math.PI * radio;

    }  
}
    

File: Circulo.py
from Figuras import Figuras

class Circulo(Figuras):

    def __init__(self, radio):
        self.radio = radio
    def area(self):
        return 3.1416 * self.radio ** 2
    def perimetro(self):
        return 2 * 3.1416 * self.radio
    def __str__(self):
        return "Circulo de radio: " + str(self.radio)